package.json

```diff
@@ -84,6 +99,7 @@
     "copy-webpack-plugin": "^8.1.1",
     "cross-env": "^7.0.3",
     "css-loader": "^5.2.0",
+    "dotenv-webpack": "^7.0.2",
```

Comment:
this is new, isn't it? where do we use env/.env configurations?

Reply:
hmmmmm yeah it seems came by a mistake